Microsoft.Toolkit.Uwp.UI.Controls/ImageEx/ImageExBase.cs

```diff
@@ -80,7 +82,7 @@ public ImageExBase()
         {
             LockObj = new object();
 
-            EffectiveViewportChanged += ImageExBase_EffectiveViewportChanged;
+            LayoutUpdated += ImageExBase_LayoutUpdated;
```

Comment:
Don't we need to do both still?

Reply:
@michael-hawker Don't need, issue #3258 is occurred by register ```EffectiveViewportChanged``` while the control host in a ```Viewbox```, it is a [framework bug](https://github.com/microsoft/microsoft-ui-xaml/issues/2389).